main.nf

```diff
@@ -48,6 +48,23 @@ process GET_SUBMISSIONS {
     """
 }
 
+// change submission status to EVALUATION_IN_PROGRESS
+process UPDATE_SUBMISSION_BEFORE_RUN {
+    secret "SYNAPSE_AUTH_TOKEN"
+    container "sagebionetworks/challengeutils:v4.2.0"
+
+    input:
+    tuple val(submission_id), val(container)
```

Comment:
There is a lot of duplicate code betwen `UPDATE_SUBMISSION_BEFORE_RUN` and `UPDATE_SUBMISSION_AFTER_RUN`.  Could we combine them into one process and use it multiples times in the workflow?

* Add `status` as an input
* Instead of `val(container)`, it can be `val(previous)` to capture the last step's output to run things in serial.

Reply:
In the latest commit, I created a module to hold the UPDATE_SUBMISSION_STATUS and then used module aliasing to import it with two different names to reuse the code in the workflow.